src/core/env/prod/terraform.tfvars

```diff
@@ -108,4 +109,20 @@ dns_ns_interop_selfcare = [
   "ns-601.awsdns-11.net",
   "ns-2030.awsdns-61.co.uk",
   "ns-119.awsdns-14.com",
-]
\ No newline at end of file
+]
+
+## Event hub
+eventhub_sku_name                 = "Standard"
+eventhub_capacity                 = 2
+eventhub_auto_inflate_enabled     = true
+eventhub_maximum_throughput_units = 4
+eventhub_zone_redundant           = true
+eventhub_alerts_enabled           = false
```

Comment:
can be `true`?

Reply:
We are just publishing into this queue, the datalake will read from it, but asked us to host it.
We have nothing to monitor here.

Reply:
OK